content/blog-posts/2023-06-20-debugging-production-ebpf-chaos/index.md

```diff
@@ -0,0 +1,9 @@
+---
+date: '2023-06-20T11:52:00.000Z'
+title: 'Debugging Production: eBPF Chaos'
+description: 'This article shares insights into learning eBPF as a new cloud-native technology which aims to improve Observability and Security workflows'
```

Comment:
did you pull the description for this article? https://www.infoq.com/articles/learning-ebpf-observability/

Reply:
I just noticed that both of the articles are actually similar at the beginning, I didn't even notice it.

- https://www.infoq.com/articles/debugging-production-ebpf-chaos/
- https://www.infoq.com/articles/debugging-production-ebpf-chaos/

Check both out

Reply:
interesting, didn't notice that